src/Handler/EnumHandler.php

```diff
@@ -44,7 +44,7 @@ public function serializeEnum(
         array $type,
         SerializationContext $context
     ) {
-        if (isset($type['params'][1]) && 'value' === $type['params'][1]) {
+        if ((isset($type['params'][1]) && 'value' === $type['params'][1]) || (!isset($type['params'][1]) && is_a($enum, \BackedEnum::class, true))) {
```

Comment:
Can `is_a($enum, \BackedEnum::class, true)` be replace with `$enum instanceof \BackedEnum`? seems much more readable to me.